### ISSUE TITLE
Is the division of the training set and the test set random?

### ISSUE THREAD

**Issue by fert-feet** [Meizhen2023/SARID#7]
Is the division of the training set and the test set random? May I ask if it is possible to re - partition the dataset? Is it recommended to do so? 
Thank you for your generous answer.
